web/app/js/components/util/Utils.js

```diff
@@ -115,6 +115,15 @@ export const toClassName = name => {
   return _lowerCase(name).replace(/[^a-zA-Z0-9]/g, "_");
 };
 
+/*
+ Create regex string from user input for a filter
+*/
+export const regexFilterString = input => {
+  input = input.replace(/[^A-Z0-9/.\-_*]/gi, "").toLowerCase();
```

Comment:
I'm not sure everywhere this is being used, but wouldn't it be easier to do something like:

```
RegExp.escape = text => text.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, '\\$&');

return new RegExp(`^.*${RegExp.escape(input)}.*`, "i");
```

Reply:
I incorporated some of this Regex feedback and will create a new ticket to test and possibly improve the Regex even more.